src/java/org/apache/cassandra/guardrails/Guardrails.java

```diff
@@ -43,201 +39,210 @@
 {
     private static final GuardrailsConfig config = DatabaseDescriptor.getGuardrailsConfig();
 
+    public static final GuardrailsFactory factory = CUSTOM_GUARDRAILS_FACTORY_PROPERTY.isPresent()
+                                                    ? CustomGuardrailsFactory.make(CUSTOM_GUARDRAILS_FACTORY_PROPERTY.getString())
```

Comment:
@jakubzytka Is there any special reason to create `CustomGuardrailsFactory.make` instead of using existing [`FBUtilities.construct`](https://github.com/datastax/cassandra/blob/ds-trunk/src/java/org/apache/cassandra/utils/FBUtilities.java#L800)?